packages/keyboard-shortcuts/src/store/actions.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Returns an action object used to register a new keyboard shortcut.
+ *
+ * @param {string} name        Shortcut name.
+ * @param {string} category    Shortcut category.
+ * @param {string} description Shortcut description.
+ * @param {Object} combination Shortcut keys combination.
```

Comment:
What's the expected shape of `{Object}` ? It might have been nice to see this new module included as covered by type-checking (#18838).

Reply:
I'll try but I might need some help :)